x-pack/plugins/observability/kibana.json

```diff
@@ -15,6 +15,7 @@
     "home",
     "lens",
     "licensing",
+    "spaces",
```

Comment:
@elastic/observability I have modified code on your side, I don't know why GH hasn't ping you, could you review it?
I just added the spaces dependency since need to use the spaces ui api in Cases.